app/controllers/admin/cabinet_ministers_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
     @whip_roles = MinisterialRole.includes(:translations).whip.order(:whip_ordering)
     @organisations = Organisation.ministerial_departments.excluding_govuk_status_closed.order(:ministerial_ordering)
 
-    render_design_system(:show, :legacy_show, next_release: false)
+    render_design_system(:show, :legacy_show, layout: get_layout)
```

Comment:
Which would mean you could ditch that third parameter altogether:

```suggestion
    render_design_system(:show, :legacy_show)
```